### ISSUE TITLE
Empty Screen instead of conversation view after 3.1.13 update

### ISSUE THREAD

**Issue by meako689** [protz/thunderbird-conversations#1502]
* **Thunderbird version:** 68.9.0 (32-bit)
* **Thunderbird Conversation version:** 3.1.13
* **Operating System:** Windows 10

**What did you do?**
Opening any email from messages list.

**What did you expect to happen?**
To see conversation view, email marked as read

**What actually happened?**
Conversation is not loaded, blank screen shown instead, email is not marked as read
![image](https://user-images.githubusercontent.com/46363/84864924-12723b00-b080-11ea-9969-6a25f0ef7d7d.png)

Error console:
![image](https://user-images.githubusercontent.com/46363/84864949-1e5dfd00-b080-11ea-847d-264655dcb46c.png)

Other addons:
none used (disabled)

Workarounds:
Disabling/enabling Convesation addon seems to resolve the issue temporarily

**Comment by Standard8** [comment]
Did this just happen only after update? Did you try restarting Thunderbird?

**Comment by meako689** [comment]
@Standard8 
No, it's not exactly after update
Tried restarting Thunderbird, uninstalling and installing add-on.

Right now seems that issue reproduces after some time of inactivity. Will try to investigate further.

**Comment by Standard8** [comment]
You might want to turn on debugging - at the bottom of Conversation's preferences.

It is a little strange, as `win.Conversations` gets set up on any mail window as it gets loaded - so by the time you select a message it should already be present.

**Comment by meako689** [comment]
Got it to be reproduced once after just restarting thunderbird. Unfortunately still not able to figure out constant conditions to reproducing.

```
WebExtensions: Loading add-on preferences from  C:\Users\Owner\AppData\Roaming\Thunderbird\Profiles\vv6hbof9.default\extensions\{d5e81018-f07b-4719-b893-3dc450504672}.xpi ext-legacy.js:190
WebExtensions: Firing profile-after-change listeners for {d5e81018-f07b-4719-b893-3dc450504672} ext-legacy.js:194
Conversations.msgWindowApi: startup api.js:257:17
TypeError: callback is not a function
msgWindowApi.js:278:7
Content Security Policy: Directive ‘child-src’ has been deprecated. Please use directive ‘worker-src’ to control workers, or directive ‘frame-src’ to control frames respectively.
Skipping ravenJS installation impala-min.js:1:43957
Conversations.msgWindowApi: Intercepted message load, 0 message(s) selected msgWindowApi.js:601:11
Failed waiting for monkeypatch to finish startup reducer.js:295:23
    checkStarted chrome://conversations/content/reducer.js:295
uncaught exception: undefined 2
Failed waiting for monkeypatch to finish startup 2 reducer.js:295:23
    checkStarted chrome://conversations/content/reducer.js:295
uncaught exception: undefined 3
Failed waiting for monkeypatch to finish startup 2 reducer.js:295:23
    checkStarted chrome://conversations/content/reducer.js:295
uncaught exception: 2147746065 autosync.jsm:251:30
Conversations.msgWindowApi: Intercepted message load, 1 message(s) selected msgWindowApi.js:601:11
win.Conversations is undefined msgWindowApi.js:411
    _summarizeThread_patched jar:file:///C:/Users/Owner/AppData/Roaming/Thunderbird/Profiles/vv6hbof9.default/extensions/gconversation@xulforum.org.xpi!/experiment-api/msgWindowApi.js:411
    _onLoad resource:///modules/SummaryFrameManager.jsm:94
    _onLoad self-hosted:1003
```

![image](https://user-images.githubusercontent.com/46363/84989002-a78e3600-b14b-11ea-944e-f0bb66045e0a.png)

**Comment by Standard8** [comment]
From that, I think I might actually know what's happening.

```
Conversations.msgWindowApi: startup api.js:257:17
TypeError: callback is not a function  msgWindowApi.js:278:7
```

msgWindowApi.js has a similarly named function to api.js, but it takes one less argument, it would seem to imply, that in some circumstances, the activities in api.js are calling the msgWindowApi.js version and failing.

It doesn't quite make sense, as I'd have thought this would cause issues with window opening and things, however, I'll try and do a patch for that later, and see how it goes.

**Comment by Standard8** [comment]
Okay, I've definitely confirmed these API experiment files can share functions, so that's likely what is causing this. I'll get a patch in the next version that should fix it.

**Comment by marmistrz** [comment]
Could you please make a new release containing this patch?

**Comment by Standard8** [comment]
I had to fix another issue first (#1501). I've at least worked around it for now, so I've just posted 3.1.14 for approval.
